### ISSUE TITLE
PL1 - Invalid lexing for statement at column 72

### ISSUE THREAD

**Issue by kkrein** [IBM/zopeneditor-about#87]
With margins set to (2,100) in the property groups for a pl1-program, the following statements:
```
 SCA0200: PROC();
  dcl v1 char(32);

 if(v1 = 'abc' ! v1 = '123') then DO ;                          
                                                             v1 = '12345678';
 end; 
 END SCA0200;
```
results in an error ("'12345678';
  
 end; 
 END SCA0200;"  is invalid lexing
) .

If the assignment to v1 is placed after column 72, the code can be parsed without an error. Is there any additional setting required to handle such cases?

**Comment by gplambert** [comment]
@kkrein This is fixed with our next release

Fixed in Z Open Editor 1.1. We extended property groups with language and compiler option properties:
https://ibm.github.io/zopeneditor-about/Docs/setting_propertygroup.html#language-scoping.

**Comment by phaumer** [comment]
Sorry, but there was a regression just for the right margin of PL/I. We already have that fixed and are waiting for the next release opportunity.

**Comment by phaumer** [comment]
We fixed this in v1.1.1. @kkrein can you try again?

**Comment by kkrein** [comment]
Hi, 
the issue seems to be resolved, 
thanks for the quick fixing!
